src/tests/eth/utils.rs

```diff
@@ -0,0 +1,150 @@
+use evm::backend::MemoryAccount;
+use primitive_types::{H160, H256, U256};
+use sha3::{Digest, Keccak256};
+use std::collections::BTreeMap;
+
+pub fn u256_to_h256(u: U256) -> H256 {
```

Comment:
Just a note, I swear that there is an `Into` trait created for `U256.into()` for `H256`..?

Reply:
utils file just copy-pusted from upstream...

Reply:
Can you check? I don't want to add to tech debt by adding in unnecessary methods if we do not need them.

Reply:
U256 doesn't contain into implementation for H160